lib/services/ProfileService.js

```diff
@@ -4,5 +4,10 @@ const Profile = require('../models/Profile');
 module.exports = class ProfileService {
   static async create({ name }) {
     // TODO: Implement me!
+    const response = await fetch(process.env.FUTURAMA_URL);
+    const [{ quote }] = await response.json();
```

Comment:
nice job leveling up the destructuring! can you imagine seeing something like this in foundations? my brain would have melted (even more)